CIPs/cip-0053.md

```diff
@@ -49,15 +49,15 @@ Remove the following elements from the BlockchainParameters contract:
 - `ClientVersion private minimumClientVersion`
 - Remove `major`, `minor`, `patch` parameters from constructor
 
-PR: https://github.com/celo-org/celo-blockchain/pull/2026
+PR: [https://github.com/celo-org/celo-blockchain/pull/2026](https://github.com/celo-org/celo-blockchain/pull/2026)
```

Comment:
Not a problem, but why did you change this? The link was showing fine before. Is this for non-github markdown viewers?

Reply:
The lintern of vscode was complaining (with warnings, but as I was checking those docs, I thought it was a good moment to modify those)